src/weights.jl

```diff
@@ -250,6 +250,76 @@ eweights(t::AbstractVector, r::AbstractRange, λ::Real) =
 
 # NOTE: No variance correction is implemented for exponential weights
 
+struct UnitWeights{S<:Real, T<:Real} <: AbstractWeights{S, T, V where V<:Vector{T}}
+    sum::S
```

Comment:
Sum is actually the length here, so maybe better call it `len`? Also, `S` can probably be replaced with just `Int` given that it represents the length of a vector.

Reply:
I replaced `sum` by `len` and added a method to the function `sum`.